src/test/java/shwxr7/model/ElementTests.java

```diff
@@ -0,0 +1,44 @@
+package shwxr7.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ElementTests {
+  @Test
+  @DisplayName("The name of an element cannot be null")
+  public void the_name_of_an_element_cannot_be_null() {
+    var badElement = Element.of(null);
+
+    assertTrue(badElement.isEmpty());
+  }
+
+  @Test
+  @DisplayName("The name of the element cannot be empty")
+  public void the_name_of_the_element_cannot_be_empty() {
+    var badElement = Element.of("");
+
+    assertTrue(badElement.isEmpty());
+  }
+
+  @Test
+  @DisplayName("One shuold be able to create an element with valid name and id")
+  public void one_should_be_able_to_create_an_element_with_valid_name_and_id() {
```

Comment:
I think, the method name should be "one_should_be_able_to_create_an_element_with_valid_name" as we have decided, that for creating an object of Element, we should provide only  the "name" of that object.

Reply:
This is true. Unfortuna I have merged the PR already. I'll fix this in a new PR.

Thanks a lot.